util/src/main/java/com/instagram/common/json/annotation/util/TypeUtils.java

```diff
@@ -209,6 +214,10 @@ public String getClassName(TypeElement type, String packageName) {
    * with a '_', i.e., the generated class for class X will be X_Y&lt;suffix&gt;.
    */
```

Comment:
nit: not your fault, we probably didn't annotate this library properly. wondering if you could add @nullable on this method.